enterprise/mqtt/build.gradle

```diff
@@ -4,6 +4,11 @@ archivesBaseName = 'mqtt'
 group = 'io.crate'
 description = 'MQTT ingestion implementation'
 
+configurations {
+    // export of test so that other modules can use the utilities there as well.
+    testOutput
+}
```

Comment:
Is this necessary? If not I would skip the importing for now. Testing modules should not be exported normally.

Reply:
We should consider adding the tests to the MQTT module and letting it depend on `ssl-impl`.